src/ISA/ISA/ArcTypes/CompositeHeader.fs

```diff
@@ -114,6 +114,23 @@ type CompositeHeader =
         | Output io             -> io.asOutput
         | FreeText str          -> str
 
+    member this.ToTerm() =
+        match this with
```

Comment:
Do we need term accessions for these headers?

Reply:
You mean for everything, which is not a CvParam?